<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>retrofitMultiMapDialog</class>
 <widget class="QDialog" name="retrofitMultiMapDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Retrofit Multi Map Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>308</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save or discard changes</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>140</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refurbishment level</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>172</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Roof refurbishment percentage</string>
   </property>
   <property name="text">
    <string>Roof %</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="roofSpinBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>168</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Roof refurbishment percentage</string>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>216</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Wall refurbishment percentage</string>
   </property>
   <property name="text">
    <string>Wall %</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="wallSpinBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>212</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Wall refurbishment percentage</string>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="windowSpinBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>252</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Window refurbishment percentage</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>256</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Window refurbishment percentage</string>
   </property>
   <property name="text">
    <string>Window %</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>retrofitMultiMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>retrofitMultiMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
